install.sh

```
@@ -499,34 +616,44 @@ download_base_config() {
+                mv "$temp_config" "$CONFIG_FILE"
+                print_message "✅ Configuration updated successfully" "$GREEN"
+                config_updated=true
+            else
+                print_message "✅ Keeping existing configuration file" "$YELLOW"
+                rm -f "$temp_config"
+            fi
         fi
     else
         mv "$temp_config" "$CONFIG_FILE"
         print_message "✅ Base configuration downloaded successfully" "$GREEN"
+        config_updated=true
```

Shellcheck (0.10.0)

[warning] config_updated appears unused. Verify use (or export if used externally).

(SC2034)